Misc/NEWS.d/next/Library/2020-03-24-16-17-20.bpo-40050.6GrOlz.rst

```diff
@@ -0,0 +1,3 @@
+Fix ``importlib._bootstrap_external``. Remove two unused imports importlib:
```

Comment:
@vstinner FYI the imports are still occurring at https://github.com/python/cpython/blob/83d46e0622d2efdf5f3bf8bf8904d0dcb55fc322/Lib/importlib/_bootstrap.py#L1168.

Reply:
Right, I know. That's why I wrote "Fix importlib._bootstrap_external."

Do you think that the NEWS entry is misleading? If yes, can you suggest a better phrasing?

Reply:
I would go with:
```
Remove redundant imports in ``importlib._bootstrap_external``.
```

Reply:
I wrote PR #19148 to change the NEWS entry.